recipes/stanhf/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "stanhf" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/stanhf-{{ version }}.tar.gz
+  sha256: 1158976afdea6dbb25ff441d0ed8531fa29b8e1dda8d29f507f7fbdc9768771d
+
+build:
+  entry_points:
+    - stanhf = stanhf.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.0
+    - pip
+  run:
+    - python >=3.8
+    - arviz
+    - matplotlib-base
+    - click
+    - click-help-colors
+    - cmdstanpy
+    - numpy
+    - pyhf
+
+test:
+  imports:
+    - stanhf
+  commands:
+    - pip check
+    - stanhf --help
+  requires:
+    - pip
+
+about:
+  license: GPL-3.0-only
+  license_file: LICENSE
```

```suggestion
  license: GPL-3.0-only
  license_file: LICENSE
  summary: stanhf
  homepage: https://github.com/xhep-lab/stanhf
  repository: https://github.com/xhep-lab/stanhf
```

Add metadata for linter